Bottom: 18234d417c39dc1d3bd2b9bb94b574c6ac443d9f
Top:    1f00f6ee1996539a85f6750c74d43b53408b42fa
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 12:43:57 +0000

Refresh of fix-3d-contour-export

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 4b81334..e183993 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -66,7 +66,7 @@
 
 #define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
-#define HACK_PLATED_BARREL_THICKNESS MM_TO_COORD(0.08) //2
+#define HACK_PLATED_BARREL_THICKNESS MM_TO_COORD(0.08)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
 #define HACK_SILK_THICKNESS MM_TO_COORD(0.01)
